lib/internal/mime.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+const {
+  Object: {
+    entries: ObjectEntries
+  },
+  MapIteratorPrototype: { next: MapIteratorPrototypeNext },
+  // RegExp,
+  RegExpPrototype: {
+    test: RegExpPrototypeTest,
+    exec: RegExpPrototypeExec
+  },
+  StringPrototype: {
+    replace: StringPrototypeReplace,
+    toLowerCase: StringPrototypeToLowerCase,
+    slice: StringPrototypeSlice
+  },
+  Symbol: {
+    replace: SymbolReplace,
+    iterator: SymbolIterator,
+  },
+  SafeMap: Map,
+} = primordials;
```

Comment:
I think this part would not land cleanly on master, you should rebase.

Reply:
Yeah, now primordials are flat:
```
const {
  ObjectEntries,
  StringPrototypeReplace,
  SymbolReplace,
  ...
} = primordials;
```

Reply:
it runs on master, but can use the flattened name